### ISSUE TITLE
Missing qpcr_records/search_record_form_error.html file

### ISSUE THREAD

**Issue by mragsac** [shassathe/covid_qPCR_test#16]
When keeping all entries blank for Advanced Search, returns an error due to a missing template file

```
[08/Apr/2020 23:56:25] "GET /qpcr_records/record_search/?csrfmiddlewaretoken=3ANTtebj4Gvpht1bVx6PX5eSBdr2rCNPkUILTBavT2CynBKdET5VC77TeSv9OGk5&barcode=1&sampling_date=&ssp_id=&sep_id=&rep_id=&rsp_id=&rwp_id=&qrp_id=&sampling_extraction_technician=&rna_extraction_technician=&qpcr_technician= HTTP/1.1" 200 14120
dict_keys(['csrfmiddlewaretoken', 'barcode', 'sampling_date', 'ssp_id', 'sep_id', 'rep_id', 'rsp_id', 'rwp_id', 'qrp_id', 'sampling_extraction_technician', 'rna_extraction_technician', 'qpcr_technician'])
Internal Server Error: /qpcr_records/record_search/
Traceback (most recent call last):
  File "/Users/mragsac/miniconda3/envs/covid_app/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/mragsac/miniconda3/envs/covid_app/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/mragsac/miniconda3/envs/covid_app/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/mragsac/miniconda3/envs/covid_app/lib/python3.8/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/mragsac/Dropbox/School/SP20_Classes/covid_qPCR_test/qpcr_records/views.py", line 371, in record_search
    return render(request, 'qpcr_records/search_record_form_error.html')
  File "/Users/mragsac/miniconda3/envs/covid_app/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/mragsac/miniconda3/envs/covid_app/lib/python3.8/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/mragsac/miniconda3/envs/covid_app/lib/python3.8/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: qpcr_records/search_record_form_error.html
```

**Comment by shassathe** [comment]
Do you have the latest push ? Not able to replicate this error

**Comment by mragsac** [comment]
@shassathe I think I have the latest version of master:

```
(covid_app) mragsac@MFR:~/Dropbox/School/SP20_Classes/covid_qPCR_test$ git pull
Already up to date.
(covid_app) mragsac@MFR:~/Dropbox/School/SP20_Classes/covid_qPCR_test$ git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   environment.yml

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	qpcr_records/migrations/0006_auto_20200408_2242.py

no changes added to commit (use "git add" and/or "git commit -a")
(covid_app) mragsac@MFR:~/Dropbox/School/SP20_Classes/covid_qPCR_test$ git log
commit 892779eb1033ebda21fffea35c52ab79186be447 (HEAD -> master, origin/master, origin/HEAD, sample_counter_dashboard)
Merge: e225979 4d5ab94
Author: Shashank Sathe <shas.sathe@gmail.com>
Date:   Wed Apr 8 16:10:35 2020 -0700

    Merge pull request #10 from shassathe/platemap

    Platemap

(covid_app) mragsac@MFR:~/Dropbox/School/SP20_Classes/covid_qPCR_test$ ls -l qpcr_records/templates/qpcr_records/ | grep error
(covid_app) mragsac@MFR:~/Dropbox/School/SP20_Classes/covid_qPCR_test$
```

**Comment by shassathe** [comment]
fixed in # 71ea9e3
